specs/projects/git_sync_write_locks/project_overview.md

```
@@ -0,0 +1,25 @@
+---
+status: complete
+---
+
+# Git Sync Write Locks
+
+
+### Context: 
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)